src/monitoring_tests/buffers_monitoring_test.py

```diff
@@ -0,0 +1,100 @@
+"""
+Checks the value of buffers every 150 settlements by invoking
+the ehtplorer api, and in some cases, coingecko.
+"""
+# pylint: disable=logging-fstring-interpolation
+import requests
+from src.monitoring_tests.base_test import BaseTest
+from src.constants import (
+    BUFFER_INTERVAL,
+    header,
+    REQUEST_TIMEOUT,
+    BUFFER_VALUE_THRESHOLD,
+)
+
+
+class BuffersMonitoringTest(BaseTest):
+    """
+    This test checks the value of the settlement contract buffers
+    every 150 settlements and generates an alert if it is higher than 200_000 USD.
+    Price feeds from ethplorer and coingecko (as backup) are used.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.counter: int = 0
+        self.buffers_value: float = 0.0
+
+    def compute_buffers_value(self) -> bool:
+        """
+        Evaluates current state of buffers.
+        """
+        # get all token balances of the smart contract
+        try:
+            resp = requests.get(
+                "https://api.ethplorer.io/\
+                    getAddressInfo/\
+                    0x9008D19f58AAbD9eD0D60971565AA8510560ab41?\
+                    apiKey=freekey",
+                headers=header,
+                timeout=REQUEST_TIMEOUT,
+            )
+            rsp = resp.json()
```

Comment:
This should probably be moved to some `EthplorerAPI`. Not sure what part of the parsing of the response can be abstracted away into the API as well.

Reply:
Would you mind if I keep this as is? I know this introduces an inconsistency but APIs that are very unlikely to be reused in any other way, such as this one, might not worth the effort properly incorporating them in the project. If you insist on this though, I can take care of it.